php/Blockonomics.php

```diff
@@ -572,6 +572,7 @@ public function get_checkout_context($order, $crypto){
                 // Display Checkout Page
                 $context['order_amount'] = $this->fix_displaying_small_values($order['satoshi']);
                 $context['payment_uri'] = $context['crypto']['uri'] . ":" . $order['address'] . "?amount=" . $context['order_amount'];
+                $context['crypto_rate'] = number_format($order['value']/$context['order_amount'], 2, '.', '');
```

Comment:
Please add code comment to note we are re-calculating the crypto_rate, which may slightly differ to rate returned by Blockonomics. Please also add reasoning to this PR as to why we are doing this, as opposed to passing this returned value with the $order.

Reply:
Done